src/Api/Models/Request/Organizations/SecretsManagerSubscriptionAddRequestModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Bit.Api.Models.Request.Organizations;
+
+public class SecretsManagerSubscriptionAddRequestModel
+{
+    [Required]
+    [Range(0, int.MaxValue)]
```

Comment:
You need at least 1 SM seat.

```suggestion
    [Range(1, int.MaxValue)]
```

Reply:
done